sources/packages/backend/apps/api/src/services/institution/institution.service.ts

```diff
@@ -717,6 +717,7 @@ export class InstitutionService extends RecordDataModelService<Institution> {
       where: {
         id: institutionId,
       },
+      cache: true,
```

Comment:
just wanted to know. is it working now? I remember you saying it was not working. was it because we didn't configure the duration in `data-source.ts` config?

Reply:
Yes exactly, you got it.